lib/hyacinth/adapters/digital_object_search_adapter/solr/document_generator.rb

```diff
@@ -48,11 +48,18 @@ def merge_title_fields_for(digital_object, solr_document = {})
         end
 
         def merge_title_fields_for!(digital_object, solr_document = {})
-          solr_document['sort_title_ssi'] = digital_object.generate_title(true)
-          indexable_title = digital_object.generate_title
-          solr_document['title_ss'] = indexable_title
-          add_keywords(indexable_title, solr_document)
-          add_titles(indexable_title, solr_document)
+          display_label = digital_object.generate_label
```

Comment:
Since label is often derived from the title, but not always, do you mind if we remove the display label stuff in here and add `'displayLabel_ss' => digital_object.generate_label` to the `merge_core_fields_for!` method?

Reply:
Actually, I just noticed that display_label is also added to the set of indexed titles.  Hmm, that does complicate things a bit.  Here's a question that we haven't answered yet: will people think of the display label AS the title when a title hasn't been assigned to the object?

Reply:
After looking at some other parts of the code below, I'm starting to think that the display_label is more of a title than the (possibly absent) title is.